packages/kit/src/views/ScanQrCode/hooks/useParseQRCode.tsx

```
@@ -39,6 +41,44 @@ import type { IToken } from '@onekeyhq/shared/types/token';
 
+export const parseOnChainAmount = async (
+  value: {
+    type: EQRCodeHandlerType;
+    data: IBaseValue;
+  },
+  token: IToken | null,
+) => {
+  const data = value.data as IChainValue;
+  if (
+    data.network &&
+    data.network.id &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)